kubernetes/config-maps/nginx/ssl.conf

```diff
@@ -0,0 +1,26 @@
+# Disable SSLv3
+ssl_protocols              TLSv1 TLSv1.1 TLSv1.2;
+
+# Enable server-side protection
+ssl_prefer_server_ciphers  on;
+ssl_ciphers                "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
+
+# Specify curve P-384
+ssl_ecdh_curve             secp384r1;
+
+# Improve ssl performence
+ssl_session_cache          shared:SSL:10m;
+ssl_session_timeout        5m;
+
+# Enable OCSP stapling
+ssl_stapling               on;
+ssl_stapling_verify        on;
+
+# Enable HSTS
+add_header Strict-Transport-Security "max-age=63072000; includeSubdomains";
+
+# Advanced settings
+server_tokens off;
+add_header X-XSS-Protection "1; mode=block";
+add_header X-Frame-Options DENY;
+add_header X-Content-Type-Options nosniff;
```

Comment:
Please add a `Referrer-Policy` header too and set it to `no-referrer`.

https://scotthelme.co.uk/a-new-security-header-referrer-policy/
https://www.perpetual-beta.org/weblog/the-curious-case-of-tls-and-the-missing-referrers.html

Reply:
Done.